webui-src/app/theme.css

```diff
@@ -327,17 +332,17 @@ a.tab-menu-item {
 }
 .tab-menu-item:hover {
   background-color: #999;
+ 
 }
+
 .tab-menu-item.selected-tab-item {
   background-color: #3ba4d7;
-  color: white;
+
 }
 
 .sidebar {
-  position: fixed;
-  height: 100%;
-  width: 200px;
-  background-color: white;
```

Comment:
Can you revert back to these changes? I'm still seeing the sidebar pushing the content panels down